codegen/src/main/kotlin/apifi/parser/models/Spec.kt

```diff
@@ -11,12 +12,18 @@ data class Path(val url: String, val operations: List<Operation>?)
 data class Operation(
         val type: PathItem.HttpMethod,
         val name: String,
-        val tags: List<String>?,
-        val params: List<Param>?,
+        val tags: List<String> = emptyList(),
+        val params: List<Param> = emptyList(),
         val request: Request?,
-        val responses: List<Response>?,
+        val responses: List<Response> = emptyList(),
         val securitySchemeType: SecurityDefinitionType = SecurityDefinitionType.BASIC_AUTH
-)
+) {
+    fun hasOnlyDefaultResponse(): Boolean = responses.size == 1 && responses.first().defaultOrStatus == ApiResponses.DEFAULT
+
+    fun hasMoreThanOne2xxResponse(): Boolean = responses.size > 1 && responses.map { it.defaultOrStatus }.all { it.startsWith("2") }
```

Comment:
@kdabir I would like to replace this `it.startsWith("2")` etc by something like `httpStatus.is2xx()`.  Currently, open-api comes with `org.apache.http`. But that module doesn't have any methods exposed for things like these. Can you suggest a thin library which can help in things like these? mapping string to status, having functions exposed on status to check if it is 2xx, 3xx etc.